src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -149,6 +149,16 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
             HelpMessage = " Whether HTTP2 is enabled.")]
         public SwitchParameter EnableHttp2 { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = " Whether FIPS is enabled.")]
+        public SwitchParameter EnableFIPS { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A list of availability zones denoting where the application gateway needs to come from.")]
+        public List<string> Zones { get; set; }
```

Comment:
This param name needs to be singular, _Zone_ (even if the user can specify multiple zones, theres always the case where they'll specify just one)

Reply:
https://github.com/Azure/azure-powershell/blob/796f80815e11eaa46ad728c09e2beda88e1de892/src/ResourceManager/Network/Commands.Network/Models/PSPublicIpAddress.cs#L38

Reply:
@akshaysngupta This is an enforced [PS guideline](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#noun-singularity). The plural name is fine in the PSObject, the corresponding param however needs to be singular.